src/NuGet.Clients/NuGet.CommandLine/Commands/ListCommand.cs

```diff
@@ -137,73 +90,55 @@ private IPackageRepository GetRepository(IEnumerable<string> listEndpoints)
             return partitioned[key: true].ToList();
         }
 
-        public override async Task ExecuteCommandAsync()
+        private IList<Configuration.PackageSource> GetEndpointsAsync()
         {
-            if (Verbose)
+            var configurationSources = SourceProvider.LoadPackageSources()
```

Comment:
>configurationSources [](start = 16, length = 20)

Can we move any of this logic down to the runner? The less that's in the entry point the better.